test-data/unit/check-tuples.test

```diff
@@ -387,7 +387,7 @@ aa, bb, *cc = t  # E: Need type annotation for variable
 [case testAssignmentToStarAnnotation]
 from typing import List
 li, lo = None, None # type: List[int], List[object]
-a, b, *c = 1, 2  # type: int, int, *List[int]
+a, b, *c = 1, 2  # type: int, int, List[int]
```

Comment:
Hm, interesting. This is nowhere spelled out in PEP 484. But given that it was always `List[int]` I think it's more correct without the star. I filed https://github.com/python/typing/issues/363 to clarify this in the PEP.